.github/workflows/llvmlite_osx-64_wheel_builder.yml

```diff
@@ -67,50 +67,24 @@ jobs:
           fi
           conda install -c "$CHAN" llvmdev=${{ env.FALLBACK_LLVMDEV_VERSION }} cmake libxml2 python-build
 
+          # Hide libunwind to prevent it from being linked against during build
```

Comment:
Can you add what happens in the case, when you do link against libuwind? For example:

```
          # Hide libunwind to prevent it from being linked against during build.
          # This prevents ... from becoming an issue.
          # See also: hyperlink
```